client/utils/errors.go

```diff
@@ -0,0 +1,9 @@
+package utils
```

Comment:
Why not in `client`?

Reply:
This whole folder needs a good cleanup...

Reply:
ahhh fair point @alessio 

@jackzampolin I don't think it's that bad imho.